articles/openshift/oshift-ref-deploy-app-into-service-mesh.md

```diff
@@ -7,7 +7,7 @@ reviewer:
 lastreviewed: 22/07/2020
 
 toc_rootlink: Reference
-toc_sub1: v4
+toc_sub1: OpenShift v4
```

Comment:
Sorry I missed this one. Could we make it "OpenShift v4.x" please. That's the format we ended up deciding on for OpenShift v3.x docs.

Reply:
Done!